### PR TITLE
Add hammer_cli_foreman_bootdisk to the hourly cron sync of git repos

### PR DESCRIPTION
**Pull request by cpeters**

### CODE REVIEW

File: .openshift/cron/hourly/git_repos [hunk 1]
```diff
@@ -40,6 +40,7 @@ update_repo foreman-selinux https://github.com/theforeman/foreman-selinux
 update_repo foreman_snapshot https://github.com/theforeman/foreman_snapshot
 update_repo foreman-tasks https://github.com/theforeman/foreman-tasks
 update_repo foreman-xen https://github.com/theforeman/foreman-xen
+update_repo hammer_cli_foreman_bootdisk https://github.com/theforeman/hammer_cli_foreman_bootdisk.git
```
Comment (domcleal) [inline .openshift/cron/hourly/git_repos]: line down, not alphabetical

**Comment by domcleal** [comment]
This and other PRs need rebasing as they conflict with your others.

**Comment by cpeters** [comment]
Updated

File: .openshift/cron/hourly/git_repos [hunk 2]
```diff
@@ -42,6 +42,8 @@ update_repo foreman_snapshot https://github.com/theforeman/foreman_snapshot
 update_repo foreman-tasks https://github.com/theforeman/foreman-tasks
 update_repo foreman-xen https://github.com/theforeman/foreman-xen
 update_repo hammer-cli-csv https://github.com/Katello/hammer-cli-csv.git
+update_repo hammer_cli_foreman_bootdisk https://github.com/theforeman/hammer_cli_foreman_bootdisk.git
+update_repo hammer-cli-foreman-discovery https://github.com/theforeman/hammer-cli-foreman-discovery
```
Comment (domcleal) [inline .openshift/cron/hourly/git_repos]: Already on line 48.

Comment (cpeters) [inline .openshift/cron/hourly/git_repos]: @domcleal I don't see it in the master branch.
https://github.com/theforeman/redmine/blob/master/.openshift/cron/hourly/git_repos#L47

Comment (domcleal) [inline .openshift/cron/hourly/git_repos]: It's there - exactly on the line you're highlighting?

Comment (cpeters) [inline .openshift/cron/hourly/git_repos]: Ok, maybe there is some confusion. This PR is to add ```hammer_cli_foreman_bootdisk```

Comment (domcleal) [inline .openshift/cron/hourly/git_repos]: Sure, but this line is adding hammer-cli-foreman-discovery - already present on line 48.

**Comment by cpeters** [comment]
@domcleal Updated to clean up PR.

**Pull request merged**